source/main.d

```diff
@@ -38,11 +38,13 @@ import std.getopt;
 int main(string[] args) @safe
 {
     ushort portNumber = 8082;
-    string[] addresses = ["::", "0.0.0.0"];
+    /* It's safer to set this to localhost and allow the user to override (not append!) */
+    static string[] defaultAddress = ["localhost"];
+    string[] cmdLineAddresses;
 
     auto opts = () @trusted {
         return getopt(args, config.bundling, "p|port", "Specific port to serve on",
-                &portNumber, "a|address", "Host address to bind to", &addresses);
+                &portNumber, "a|address", "Host address to bind to", &cmdLineAddresses);
```

Comment:
Shouldn't that be `Host address_es_` then? I think it would also be good to print the default values on `--help`

Reply:
> Shouldn't that be `Host address_es_` then? I think it would also be good to print the default values on `--help`

I'm deliberately using defaultAddress (singular) and cmdLineAddresses (plural) to make the ternary expression below more readable at a glance and to reinforce that it's possible to add more than one listening address on the cmdLine.

Reply:
FTR, the syntax is `<service> -a someaddress -a someotheraddress` when adding more than one listening address.

Ikey is quite keen on help being readable on 80 columns wide terminals (as he's using one!), so any help text update you decide to PR needs to take that into account.